src/client/user/actions/types.ts

```diff
@@ -6,49 +6,52 @@ import {
 } from '../reducers/types'
 
 /* User actions */
-export const CREATE_URL_SUCCESS = 'CREATE_URL_SUCCESS'
-export const GET_URLS_FOR_USER_SUCCESS = 'GET_URLS_FOR_USER_SUCCESS'
-export const OPEN_CREATE_URL_MODAL = 'OPEN_CREATE_URL_MODAL'
-export const CLOSE_CREATE_URL_MODAL = 'CLOSE_CREATE_URL_MODAL'
-export const SET_SHORT_URL = 'SET_SHORT_URL'
-export const SET_LONG_URL = 'SET_LONG_URL'
-export const SET_EDITED_LONG_URL = 'SET_EDITED_LONG_URL'
-export const SET_RANDOM_SHORT_URL = 'SET_RANDOM_SHORT_URL'
-export const RESET_USER_STATE = 'RESET_USER_STATE'
-export const TOGGLE_URL_STATE_SUCCESS = 'TOGGLE_URL_STATE_SUCCESS'
-export const SET_URL_TABLE_CONFIG = 'SET_URL_TABLE_CONFIG'
-export const SET_URL_FILTER = 'SET_URL_FILTER'
-export const UPDATE_URL_COUNT = 'UPDATE_URL_COUNT'
-export const IS_FETCHING_URLS = 'IS_FETCHING_URLS'
-export const WIPE_USER_STATE = 'WIPE_USER_STATE'
-export const SET_IS_UPLOADING = 'SET_IS_UPLOADING'
-export const SET_UPLOAD_FILE_ERROR = 'SET_UPLOAD_FILE_ERROR'
-export const SET_CREATE_SHORT_LINK_ERROR = 'SET_CREATE_SHORT_LINK_ERROR'
-export const SET_LAST_CREATED_LINK = 'SET_LAST_CREATED_LINK'
-export const SET_EDITED_CONTACT_EMAIL = 'SET_EDITED_CONTACT_EMAIL'
-export const SET_EDITED_DESCRIPTION = 'SET_EDITED_DESCRIPTION'
-export const SET_USER_MESSAGE = 'SET_USER_MESSAGE'
-export const SET_USER_ANNOUNCEMENT = 'SET_USER_ANNOUNCEMENT'
-export const SET_URL_UPLOAD_STATE = 'SET_URL_UPLOAD_STATE'
-export const SET_FILE_UPLOAD_STATE = 'SET_FILE_UPLOAD_STATE'
+
+export enum UserAction {
+  CREATE_URL_SUCCESS = 'CREATE_URL_SUCCESS',
+  GET_URLS_FOR_USER_SUCCESS = 'GET_URLS_FOR_USER_SUCCESS',
+  OPEN_CREATE_URL_MODAL = 'OPEN_CREATE_URL_MODAL',
+  CLOSE_CREATE_URL_MODAL = 'CLOSE_CREATE_URL_MODAL',
+  SET_SHORT_URL = 'SET_SHORT_URL',
+  SET_LONG_URL = 'SET_LONG_URL',
+  SET_EDITED_LONG_URL = 'SET_EDITED_LONG_URL',
+  SET_RANDOM_SHORT_URL = 'SET_RANDOM_SHORT_URL',
+  RESET_USER_STATE = 'RESET_USER_STATE',
+  TOGGLE_URL_STATE_SUCCESS = 'TOGGLE_URL_STATE_SUCCESS',
+  SET_URL_TABLE_CONFIG = 'SET_URL_TABLE_CONFIG',
+  SET_URL_FILTER = 'SET_URL_FILTER',
+  UPDATE_URL_COUNT = 'UPDATE_URL_COUNT',
+  IS_FETCHING_URLS = 'IS_FETCHING_URLS',
+  WIPE_USER_STATE = 'WIPE_USER_STATE',
+  SET_IS_UPLOADING = 'SET_IS_UPLOADING',
+  SET_UPLOAD_FILE_ERROR = 'SET_UPLOAD_FILE_ERROR',
+  SET_CREATE_SHORT_LINK_ERROR = 'SET_CREATE_SHORT_LINK_ERROR',
+  SET_LAST_CREATED_LINK = 'SET_LAST_CREATED_LINK',
+  SET_EDITED_CONTACT_EMAIL = 'SET_EDITED_CONTACT_EMAIL',
+  SET_EDITED_DESCRIPTION = 'SET_EDITED_DESCRIPTION',
+  SET_USER_MESSAGE = 'SET_USER_MESSAGE',
+  SET_USER_ANNOUNCEMENT = 'SET_USER_ANNOUNCEMENT',
+  SET_URL_UPLOAD_STATE = 'SET_URL_UPLOAD_STATE',
+  SET_FILE_UPLOAD_STATE = 'SET_FILE_UPLOAD_STATE',
+}
 
 export type SetUrlUploadStateAction = {
-  type: typeof SET_URL_UPLOAD_STATE
+  type: typeof UserAction.SET_URL_UPLOAD_STATE
```

Comment:
```suggestion
  type: UserAction.SET_URL_UPLOAD_STATE
```

I think we can also take the opportunity to narrow the type even further for all these actions, as the corresponding action will always have that exact string value and no other.